src/main/java/Parser.java

```diff
@@ -0,0 +1,206 @@
+import javafx.application.Platform;
+
+public class Parser {
+
+    /** The number of tasks at the start of the program. */
+    public static int totalTasks = 0;
+
+    /**
+     * Process the user input to make sense for the system.
+     * @param nextInput The user input.
+     * @param tasks The Task Arraylist containing user tasks in sequence.
+     * @param ui UI structure to show the user correct message.
+     * @return Return the correct logic and GUI output.
+     */
+    public String processInput(String nextInput, TaskList tasks, Ui ui) {
+        String command = nextInput.contains(" ") ? nextInput.split(" ")[0] : nextInput;
+        try {
+            switch (command) {
+                case "todo":
+                    return todo(nextInput, tasks, ui);
+                case "deadline":
+                    return deadline(nextInput, tasks, ui);
+                case "event":
+                    return event(nextInput, tasks, ui);
+                case "done":
+                    return done(nextInput, tasks, totalTasks, ui);
+                case "delete":
+                    return delete(nextInput, tasks, totalTasks, ui);
+                case "list":
+                    return list(tasks, ui);
+                case "find":
+                    return find(nextInput, tasks, ui);
+                case "exit":
+                    return exitCommand();
+                default:
+                    return wrongCommand();
+            }
```

Comment:
Hmm, switch case does not seem to follow the coding standard. The 'case' should be indented at the same level as 'switch'! :)